src/rdkafka_cgrp.h

```diff
@@ -231,8 +231,22 @@ typedef struct rd_kafka_cgrp_s {
         /** If a SUBSCRIBE op is received during a COOPERATIVE rebalance,
          *  actioning this will be postponed until after the rebalance
          *  completes. The waiting subscription is stored here.
-         *  Mutually exclusive with rkcg_next_subscription. */
+         *  Mutually exclusive with rkcg_next_subscription.
+         *
+         *  For the consumer protocol, this field doesn't include regex
+         *  subscriptions. For that please refer `rkcg_subscription_regex` */
```

Comment:
This doc if for `rkcg_next_subscription`, the field should include regex subscriptions.
Move it to the doc for `rkcg_subscription_topics`